src/components/Content.tsx

```diff
@@ -0,0 +1,5 @@
+import { Outlet } from 'react-router-dom';
+
+export const Content = () => {
+  return <Outlet />;
+};
```

Comment:
What is the point of this component?
It's better to create a `Layout` component where you can move your navigation and use it as a wrapper for all pages that have this navigation
https://medium.com/@vitorbritto/react-design-patterns-layout-components-pattern-455c98e0bf92